eos/b-decays/lifetime.cc

```diff
@@ -272,6 +273,190 @@ namespace eos
+                    -60.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -144.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    768.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z))
+                },
+                {
+                    -6.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -60.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -3.0 / 2.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    0,
+                    -60.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                },
+                {
+                    -36.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -144.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    0,
+                    -36.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    576.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                },
+                {
+                    336.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    768.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -60.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    576.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    -12480.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                }
+            }};
+
+            const std::array<std::complex<double>, 5u> C_sing
+            {
+                wc.c1(),  wc.c3(),  wc.c5(),  wc.c7(),  wc.c9()
+            };
+
+            std::array<std::complex<double>, 5u> Cconj_sing;
+            complex<double> (*conj_sing)(const std::complex<double> &) = &std::conj<double>;
+            std::transform(C_sing.cbegin(), C_sing.cend(), Cconj_sing.begin(), conj_sing);
```

Comment:
could you not replace ``conj_sing`` with ``&std::conj<double>`` in line 331 and remove line 330 entirely?